catalog/utilities/media_props_gen/docs/media_props.md

```diff
@@ -2,424 +2,376 @@
 
-## Object Shape
+- Empty tags (`{}`) are filtered out, see
+  [cc-archive issue](https://github.com/cc-archive/cccatalog-api/issues/130)
+- Tags with license slugs are removed because they often don't match the media
+  item license. These tags, as well as some other deny-listed tags are filtered
+  out.
+- Some machine-generated tags have accuracy lower than 90% and are unreliable.
+  These tags are filtered out.
 
-## Selection Criteria
+Some inconsistencies are not fixed by the cleanup process:
 
-## Normalization and Validation
+- Incorrectly encoded tags, see
+  [issue #1927](https://github.com/WordPress/openverse/issues/1927). This can
+  result in duplicate tags when the frontend decodes the tags.
+- Tags with leading or trailing spaces, see
+  [issue 4199](https://github.com/WordPress/openverse/issues/4199)
 
-## Existing Data Inconsistencies
+Identical, duplicate tags were filtered out in
+[#1556](https://github.com/WordPress/openverse/issues/1566)
 
-# last_synced_with_source
+# watermarked
 
 ## Description
 
-## Object Shape
+Whether the image has a discernible watermark. If this field is null or false,
+it does not mean the image doesn't have a watermark. This field was set to true
+for some CommonCrawl providers (McCord Museum, 500px, FloraOn, IHA). Currently,
+no provider script or SQL ingestion sets this field value.
 
-## Selection Criteria
+# last_synced_with_source
 
-## Normalization and Validation
+## Description
 
-## Existing Data Inconsistencies
+For new items, the timestamp that is the same as the `created_on`. For items
+that were updated during re-ingestion, the timestamp of re-ingestion.
 
 # removed_from_source
 
 ## Description
 
-## Object Shape
+Set to `True` for items that were not updated during re-run of the provider
+script. Items that have `True` in `removed_from_source` are not added to the ES
+index during the data refresh process.
 
 ## Selection Criteria
 
-## Normalization and Validation
-
-## Existing Data Inconsistencies
+[`expire_old_images`](https://github.com/WordPress/openverse/blob/main/catalog/dags/retired/common/loader/sql.py)
+DAG added in
+[Expiration of outdated images in the database](https://github.com/cc-archive/cccatalog/pull/483)
+was used to set `removed_from_source` to `True` for images that were updated
+more than `OLDEST_PER_PROVIDER` value.
 
 # filetype
 
 ## Description
 
+The filetype of the media item (not the MIME type). If the filetype is not
+available in the API response, it can be extracted from the URL extension or
+from the HEAD response from the media direct URL.
+
 ## Object Shape
 
-## Selection Criteria
+String, the file type (extension) of the audio file. Maximum length: 5
+characters.
 
 ## Normalization and Validation
 
-## Existing Data Inconsistencies
+`catalog/dags/common/extensions.py` The `get_file_extension` function is used to
+get the file extension from a URL. The function returns the file extension in
```

Comment:
Do you mind linking to code here, even if (similar to the above) it uses a reference to `main` rather than a specific commit?